ui/nuxeo-path-suggestion/nuxeo-path-suggestion.js

```diff
@@ -173,6 +174,23 @@ import { FormatBehavior } from '../nuxeo-format-behavior.js';
       };
     }
 
+    attached() {
```

Comment:
Academic question: Why are we using here `attached()` instead of `ready()`?

Reply:
Because [this.$.typeahead](https://github.com/nuxeo/nuxeo-elements/pull/348/files/8b1189453366a6e16ac97709a5b60033e691695c#diff-275bb7100f70990da7d0e3ef3dca5c78e1cacf2184a48b0ed18cdd670e7a6c0eR191) not yet initialized in `ready()`. It somehow contradicts the Polymer documentation for [element lifecycle](https://polymer-library.polymer-project.org/1.0/docs/devguide/registering-elements#lifecycle-callbacks)🤷 I was a bit confused by it